presto-orc/src/test/java/com/facebook/presto/orc/OrcTester.java

```diff
@@ -478,28 +485,75 @@ private void testRoundTripType(Type type, List<?> readValues)
 
     public void testRoundTripTypes(List<Type> types, List<List<?>> readValues, List<Map<Integer, Map<Subfield, TupleDomainFilter>>> filters)
             throws Exception
+    {
+        testRoundTripTypes(types, readValues, filters, Optional.empty());
+    }
+
+    public void testRoundTripTypes(List<Type> types, List<List<?>> readValues, List<Map<Integer, Map<Subfield, TupleDomainFilter>>> filters, Optional<List> order)
```

Comment:
`filters` is a list where each element is a set of filters to use in a single test run; multiple elements trigger multiple runs; hence, `order` must be a list of the same size where i-th element provides expected order for the i-th set of filters

Reply:
You're saying order should be a list of lists. We should not count on filters being a list of size 1. And order[i] should match the order of filters[i]. Is that correct?

Reply:
That's correct. Also, see https://github.com/prestodb/presto/pull/13430 I think it would be best to add this to the new OrcReaderSettings.